lib/Elastica/ResultSet/ProcessingBuilder.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Elastica\ResultSet;
+
+use Elastica\Query;
+use Elastica\Response;
+use Elastica\ResultSet;
+
+class ProcessingBuilder implements BuilderInterface
```

Comment:
Since this is actually a decorator for a normal Builder instance, i wouldnt be calling this one a Default or Base - it is designed to wrap another builder regardless of the implementation.